application/models/services/QuestionAggregateService/SubQuestionsService.php

```diff
@@ -65,7 +65,9 @@ public function save(Question $question, $subquestions)
      */
     public function delete($surveyId, $subquestionId)
     {
+        $survey = \Survey::model()->findByPk($surveyId);
         if (
+            $survey->isActive ||
             !\Permission::model()->hasSurveyPermission(
```

Comment:
Permission model should also have been injected here not sure what task that from.

Reply:
fixed